packages/tools/guiEditor/src/components/commandBarComponent.tsx

```diff
@@ -96,6 +99,11 @@ export class CommandBarComponent extends React.Component<ICommandBarComponentPro
         const copyyIcon = this.props.globalState.selectedControls.length === 0 ? copyIconDisabled : copyIcon;
         const deleteeIcon = this.props.globalState.selectedControls.length === 0 ? deleteIconDisabled : deleteIcon;
         const pasteeIcon = isPasteDisabled ? pasteIconDisabled : pasteIcon;
+
+        const responsiveSelected = this.props.globalState.fromPG ? window.localStorage.getItem("responsive") === "true" : DataStorage.ReadBoolean("Responsive", true);
+        const responsiveUI = this.props.globalState.fromPG ? window.localStorage.getItem("responsiveUI") === "true" : DataStorage.ReadBoolean("Responsive", true);
+        const unresponsiveUI = this.props.globalState.fromPG ? window.localStorage.getItem("responsiveUI") === "false" : !DataStorage.ReadBoolean("Responsive", true);
+

```

Comment:
It is! I guess I just did it this way because breaking it down to this level was what made the most sense to me when thinking about the logic, but I can also change it since it is a bit repetitive